ansible/files/telemetry.sh

```diff
@@ -48,6 +48,9 @@ fi
 PRESTO_CONFIG_SCRIPT=$(cat <<EOF
 while ! pgrep presto > /dev/null; do sleep 1; done
 
+# install presto plugins
+sudo -u presto aws s3 sync $TELEMETRY_CONF_BUCKET/plugins/ /usr/lib/presto/plugin/
```

Comment:
Is there some code that updates the plugin on S3 when a new version is pushed? We are probably going to add more plugins in the future and it would be nice if there was some way to automate this.